### PR TITLE
SME-16 - infinite scroller in content page / main page

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: pwa-shell/src/state/content/selectors.js [hunk 1]
```diff
@@ -10,6 +10,17 @@ export const getContentList = createSelector(
...
 );
 
+export const getContentListMeta = createSelector(
+  getContent,
+  content => content.listMeta,
+);
+
+export const getListHasMorePages = createSelector(
+  getContentListMeta,
+  // meta => !!Object.keys(meta).length && meta.page < meta.lastPage,
```
Comment (gidesan) [inline pwa-shell/src/state/content/selectors.js]: please avoid commented code

File: pwa-shell/src/ui/content-list/ContentListContainer.js [hunk 2]
```diff
@@ -12,25 +14,38 @@ import {
...
     const { contentType } = ownProps.match.params;
-    dispatch(fetchContentListByContentType(contentType));
-  }
+    dispatch(fetchContentListByContentType(contentType, pageObj));
+  },
+});
+
+const getNextPageMeta = ({ page, pageSize }) => ({ page: page + 1, pageSize });
+
+export const mergeProps = (stateProps, dispatchProps) => ({
```
Comment (gidesan) [inline pwa-shell/src/ui/content-list/ContentListContainer.js]: please avoid `mergeProps` when possible: put logic in components instead

File: pwa-shell/src/ui/content-list/ContentList.js [hunk 3]
```diff
@@ -60,21 +63,24 @@ class ContentList extends PureComponent {
...
-            : 'Nessun argomento selezionato. Seleziona almeno un argomento dal menu in alto a sinistra.'
-        }
+        <Container fluid className="content">
+          {
+            selectedCategoryCodes && selectedCategoryCodes.length
+              ? (
+                contentList && contentList.length ?
+                <InfiniteScroll
+                  pageStart={0}
+                  loadMore={fetchNextContentList}
```
Comment (gidesan) [inline pwa-shell/src/ui/content-list/ContentList.js]: for a better readability, what about renaming `fetchNextContentList` to `loadMoreListItems`?

**Comment by jeffgo10** [comment]
changes/fixes applied. please review

File: pwa-shell/src/ui/content-list/ContentList.js (hunk 3 above)
Comment (jeffgo10) [reply pwa-shell/src/ui/content-list/ContentList.js]: changed and moved to the component file

**Pull request merged**